README.rst

```diff
@@ -64,15 +70,28 @@ key to download a request using Zyte API. Full list of parameters is provided in
 
 .. code-block:: python
 
-    yield scrapy.Request(
-        "http://books.toscrape.com/",
-        callback=self.parse,
-        meta={
-            "zyte_api": {
-                "browserHtml": True,
-                "geolocation": "US",
-                "javascript": True,
-                "echoData": {"something": True}
-            }
-        }
-    )
+   import scrapy
+
+   class TestSpider(scrapy.Spider):
+       name = 'test'
+       start_urls = ['http://books.toscrape.com/']
+
+       def start_requests(self):
+
+               yield scrapy.Request(url="http://books.toscrape.com/", callback=self.parse,
+                                    meta={
+                                        "zyte_api": {
+                                            "browserHtml": True,
+                                            # You can set any GEOLocation region you want.
+                                            "geolocation": "US",
+                                            "javascript": True,
+                                            "echoData": {"something": True}
+                                        }
+                                    })
+
+       def parse(self, response, **kwargs):
```

Comment:
`**kwargs` can be removed here.

Reply:
Yup, removed that as well. and added a new Commit.